files/tasks/checkout_packit.yaml

```diff
@@ -0,0 +1,28 @@
+---
+- set_fact:
+    reverse_dir: "{{ project_dir }}/rev_project_cloned"
+
+- name: Install ansible
+  dnf:
+    name:
+    - ansible
+    state: present
+  become: true
+- name: Install flexmock
```

Comment:
probably not important here, because packit not using tox for testing, so that dep is already installed. (in requre dep dep testing via ogr, it installed flexmock to virtenv)
so probably you can remove this.

Reply:
Thanks, good to know.